rotkehlchen/chain/ethereum/adex/adex.py

```diff
@@ -322,38 +399,46 @@ def _deserialize_channel_withdraw(
         """Deserialize a channel withdraw event. Only for Tom pool.
 
         It may raise KeyError.
-
-        TODO: use bidict or already pass the inverse identity_address_map.
         """
```

Comment:
Docstring says It may raise a KeyError? I see you also raising DeserializationError. Please add it to docstring and make sure you handle it

Reply:
It was an old idea of implementing a bidict for the user_identity:EOA map, as keys and values will be unique. I wouldn't need to invert it as a I do in two places.

Reply:
Hmmm you did not read my comment? I don't care about the removal. Read what I wrote. It just shows at that line since this is how github renders it.

Reply:
Apologies. Your comment has been useful for fixing an error handling.